src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push({
+        id: Math.floor(Math.random() * 100),
+        title: action.payload.title,
+        author: action.payload.author,
+      });
```

Comment:
[OPTIONAL]
- [ ] AS said in the previous review, your state needs to be immutable i.e cannot be modified that is why the previous reviewer suggested those changes, however since it's required for this milestone you can move forward but in the next once you learn about immutability that's the way you will be implementing your reducer all the time:
```jsx
addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
```